doc/base/classes.xml

```diff
@@ -12293,7 +12293,8 @@
 			</argument>
 			<description>
 				Add a custom property info to a property. The dictionary must contain: name:[String](the name of the property) and type:[int](see TYPE_* in [@Global Scope]), and optionally hint:[int](see PROPERTY_HINT_* in [@Global Scope]), hint_string:[String].
-			Example:[codeblock]
+			Example:
```

Comment:
The whole block should be indented one tab further to match the "Add a custom property info..." part in theory, would you be able to amend the commit for that?